module-4/metascan/README.md

```diff
@@ -82,13 +81,3 @@ python3 metascan.py -u https://example.com -n example_metadata.csv
 - `requests`: For making HTTP requests to fetch web pages.
 - `beautifulsoup4`: For parsing HTML and extracting data from web pages.
 - `PyPDF2`: For extracting metadata from PDF files.
-
-## Notes
-
-- Ensure the website allows scraping of PDF links and complies with legal and ethical guidelines.
-- The script may encounter issues with PDF files that have restricted access or malformed metadata.
-- Customize the script further based on specific PDF metadata extraction needs or website structures.
-
-## License
-
-This project is licensed under the MIT License - see the LICENSE file for details.
```

Comment:
Why delete this section, instead of adding a link to the file?

Reply:
i dont have a license